src/plugins/integrations/AWSIntegration.ts

```diff
@@ -82,14 +82,15 @@ class AWSIntegration implements Integration {
           const parentSpan = tracer.getActiveSpan();
           const originalCallback = callback;
 
+          request.params = request.params ? request.params : {};
+
           let activeSpan: ThundraSpan = null;
 
           const operationName = request.operation ? request.operation  : AWS_SERVICE_REQUEST;
 
           if (serviceName === 'sqs') {
             const operationType = SQSRequestTypes[operationName];
-            let queueName = operationType ?
-            Utils.getQueueName(request.params.QueueUrl) : AWS_SERVICE_REQUEST;
+            let queueName = koalas(Utils.getQueueName(request.params.QueueUrl), AWS_SERVICE_REQUEST);
```

Comment:
If queue name is not valid for the request, it is ok to use `AWS_SERVICE_REQUEST ` as the default span name, but it should not be used for `AwsSQSTags.QUEUE_NAME` tag. So you may handle this by introducing another local variable for span name.

Reply:
Done. Introduced a `spanName` variable.